src/Compiler/test/Microsoft.AspNetCore.Razor.Test.ComponentShim/Microsoft.AspNetCore.Components.Web.netstandard2.0.cs

```diff
@@ -370,6 +370,13 @@ public partial class ProgressEventArgs : System.EventArgs
         public long Total { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public string Type { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
     }
+    public static partial class RenderMode
+    {
+        public static ServerRenderMode Server { get; } = new();
```

Comment:
I think the `new();` is unnecessary in shims.

Reply:
We were testing the type returned by the attribute. We can't easily do that if it's null, so I'll keep it as an instance as I think its valuable in the test.